Cargo.toml

```diff
@@ -23,9 +24,11 @@ wasmtime-environ = { version = ">=15.0.1", features = [ "component-model" ] }
 wit-component = { version = ">=0.19.0", default-features = false }
 wit-parser = { version = ">=0.13.0", default-features = false }
 
+tracing = { version = "0.1" }
```

Comment:
Will of course clean this one up when it is working.

tracing is such a monumental helper for the web to inspect state when you don't have a debugger